cmd/create/cluster/cmd.go

```diff
@@ -1016,80 +1009,76 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	// Errors when users elects for cluster admin via flags and elects for hosted control plane via interactive prompt"
-	if isHostedCP && (createAdminUser || clusterAdminPassword != "") {
-		r.Reporter.Errorf("Setting Cluster Admin is only supported in classic ROSA clusters")
-		os.Exit(1)
-	}
-
+	createAdminUser := args.createAdminUser
+	clusterAdminUser := admin.ClusterAdminUsername //strings.Trim(args.clusterAdminUser, " \t")
```

Comment:
@marcolan018 Shall we remove the `//strings.Trim` comment if it's not being used?

Reply:
@robpblake removed the comments